elixir/apps/web/assets/css/app.css

```diff
@@ -1,3 +1,4 @@
 @import "tailwindcss/base";
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
+@import "./scrollbar.css"
```

Comment:
A newline at the end of file is needed by our conventions.

Reply:
Might need a semicolon at the end here too.

Reply:
Sorry for the confusion -- there should be _no_ newline here.